tasks/Ubuntu.yml

```diff
@@ -78,23 +67,23 @@
     loop_var: pkg
   tags: install
 
+- name: "Setup repository key for OpenStack repository"
+  apt_key:
+    id: 5EDB1B62EC4926EA
+    keyserver: keyserver.ubuntu.com
+  when:
+    - openio_repository_manage_openstack_repository
+    #- openio_repository_openstack_release == 'train'
+
 - name: "Configure OpenStack {{ openio_repository_openstack_release }} release repository"
   apt_repository:
     repo: "deb http://ubuntu-cloud.archive.canonical.com/ubuntu {{ ansible_lsb.codename }}-updates/{{ openio_repository_openstack_release }} main"
+    update_cache: false
```

Comment:
why is that needed now ? Is this a bugfix ?

Reply:
Yes. 
The task `name: "Update APT cache"` is here for that.